### PR TITLE
Ofmcc 5753 supp apps UI update

### PR DESCRIPTION
**Pull request by jenbeckett**
- Added UI categories for ticket https://eccbc.atlassian.net/browse/OFMCC-5753

- Additionally - did a bit of refactoring to add PANELS objects into a constants file. With that - I also changed the logic on Supplementary Submit Page to check the models for completed applications - instead of adding that information to the panel object 

- Small code changes to update props to snake-case and other linter recommendations

### CODE REVIEW

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 1]
```diff
@@ -75,32 +79,48 @@
...
           </v-expansion-panel-text>
         </v-expansion-panel>
       </v-expansion-panels>
     </div>
+
     <div v-if="nextTermActive">
       <v-skeleton-loader v-if="loading" :loading="loading" type="table-tbody"></v-skeleton-loader>
       <v-expansion-panels v-else v-model="panel" multiple>
-        <v-expansion-panel v-for="panel in PANELS" :key="panel.id" :value="panel.id">
+        <v-expansion-panel-title class="tooltip py-6" hide-actions>
```
Comment (weskubo-cgi) [inline frontend/src/views/supp-allowances/SupplementaryFormView.vue:119]: See previous comment about usage of v-expansion-panel.

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 2]
```diff
@@ -109,24 +129,35 @@
...
             <SupportNeedsProgrammingAllowance
               v-if="panel.id === SUPPORT_NEEDS"
               :supportModel="getModel(SUPPLEMENTARY_TYPES.SUPPORT, nextRenewalTerm)"
               :hasInclusionPolicy="currentOrg.hasInclusionPolicy"
               :formDisabled="formDisabled"
               @update="updateModel" />
+          </v-expansion-panel-text>
+        </v-expansion-panel>
+
+        <v-expansion-panel-title class="tooltip py-6 mt-8" hide-actions>
```
Comment (weskubo-cgi) [inline frontend/src/views/supp-allowances/SupplementaryFormView.vue:143]: See previous comment about usage of v-expansion-panel.

File: frontend/src/views/supp-allowances/SupplementarySubmitView.vue [hunk 3]
```diff
@@ -13,7 +13,7 @@
         </AppButton>
```
Comment (weskubo-cgi) [inline frontend/src/views/supp-allowances/SupplementarySubmitView.vue:13]: Most of the comments on panels apply to this component as well.

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 4]
```diff
@@ -66,7 +66,11 @@
     <div v-if="!nextTermActive">
       <v-skeleton-loader v-if="loading" :loading="loading" type="table-tbody"></v-skeleton-loader>
       <v-expansion-panels v-else v-model="panel" multiple>
-        <v-expansion-panel v-for="panel in PANELS" :key="panel.id" :value="panel.id">
+        <v-expansion-panel-title class="tooltip py-6" hide-actions>
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:69]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 5]
```diff
@@ -75,32 +79,48 @@
...
             <SupportNeedsProgrammingAllowance
               v-if="panel.id === SUPPORT_NEEDS"
               :supportModel="getModel(SUPPLEMENTARY_TYPES.SUPPORT, renewalTerm)"
               :hasInclusionPolicy="currentOrg.hasInclusionPolicy"
               :formDisabled="currentTermDisabled || formDisabled"
               @update="updateModel" />
+          </v-expansion-panel-text>
+        </v-expansion-panel>
+
+        <v-expansion-panel-title class="tooltip py-6 mt-8" hide-actions>
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:93]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 6]
```diff
@@ -75,32 +79,48 @@
...
+          <span class="header-label">Discretionary Services Allowance</span>
+        </v-expansion-panel-title>
+        <v-expansion-panel :key="DISCRETIONARY_PANEL.id" :value="DISCRETIONARY_PANEL.id">
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:96]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue (hunk 1 above)
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:119]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue (hunk 2 above)
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue:143]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue [hunk 7]
```diff
@@ -66,7 +66,11 @@
     <div v-if="!nextTermActive">
       <v-skeleton-loader v-if="loading" :loading="loading" type="table-tbody"></v-skeleton-loader>
       <v-expansion-panels v-else v-model="panel" multiple>
-        <v-expansion-panel v-for="panel in PANELS" :key="panel.id" :value="panel.id">
+        <v-expansion-panel-title class="tooltip py-6" hide-actions>
+          <span class="header-label">Core Services Allowance</span>
+        </v-expansion-panel-title>
+
+        <v-expansion-panel v-for="panel in CORE_SERVICES_PANELS" :key="panel.id" :value="panel.id">
```
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue]: done

File: frontend/src/views/supp-allowances/SupplementaryFormView.vue (hunk 4 above)
[the remark this answers is not in the document]
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementaryFormView.vue]: done

File: frontend/src/views/supp-allowances/SupplementarySubmitView.vue (hunk 3 above)
Comment (jenbeckett) [reply frontend/src/views/supp-allowances/SupplementarySubmitView.vue:13]: done

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**